test/regressions/tests/SpeedDial/Directions.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Avatar from '@material-ui/core/Avatar';
 import { withStyles } from '@material-ui/core/styles';
-import { capitalize } from '@material-ui/core/utils';
+import { capitalize } from '@material-ui/utils';
```

Comment:
What do you think about keeping the same imports as before but simply having core re-export utils? It would:

- remove the need for a codemod
- remove the need for developers to import from the utils package
- solve the release fail
- reduce the git diff

```suggestion
import { capitalize } from '@material-ui/core/utils';
```

Reply:
Yeah I think that is much safer option :) will probably open new PR with only these changes